src/components/TradeHistory.js

```diff
@@ -101,7 +101,8 @@ const TradingHistory = observer(() => {
                 <TableHeader className="align-right">Time</TableHeader>
             </TableHeadersWrapper>
             <TableRowsWrapper>
-            {recentTrades.map((trade, i) => (
+            {recentTrades.map((trade, i) =>
+              (trade && trade.type) ? (
```

Comment:
I'm curious when we get invalid trade types?

Reply:
I got an error sometimes when using kovan, thats why Im adding this extra check.